config/karma/shared.karma.conf.js

```diff
@@ -67,8 +71,16 @@ function getConfig(config) {
       {
         pattern: specStyles,
         watched: false
+      },
+      {
+        pattern: assetsPattern,
+        included: false,
+        served: true,
       }
     ],
+    proxies: {
+      '/~/': `/absolute${srcPath}`
```

Comment:
I added an image to the template and ran `skyux test`, got this warning:
```
WARN [proxy]: failed to proxy /absolute/Users/stevebr/Projects/github/blackbaud/skyux-sdk-template/src/assets/my-image.jpg (socket hang up)
```

Reply:
Thanks Steve.  Just following up, there does appear to be a race condition where a test will finish before the image proxy is done.  I was unable to replicate this consistently, but am confident these changes are an improvement over the 404.

As discussed, we're going to proceed with merging this.